MiniApp/Classes/MiniApp.swift

```diff
@@ -5,36 +5,39 @@ public class MiniApp: NSObject {
     /// Error information will be returned if any problem while fetching from the backed
     ///
     /// - Parameters:
+    ///     -   config: on optional custom configuration to override default SDK parameters such as base URL, app ID, subsciption key or host app version
```

Comment:
In the public API documentation, I think it will be good to specify the use cases when to use this optional config e.g. QA (it gives SDK authors a good chance to specify the intended use as well).